lnbits/core/views/api.py

```diff
@@ -378,6 +397,8 @@ async def payment_received() -> None:
 async def api_payments_sse(
     request: Request, wallet: WalletTypeInfo = Depends(get_key_type)
 ):
+    # FIXME: wallet.wallet can be None here
+    assert wallet.wallet is not None
```

Comment:
Do you need these?

Reply:
ye, wallet can be None here, if we do not assert the code it will break anyway, there are 2 solutions to the problem.

1. we try to make the wallet type Wallet, not Optional[Wallet]
2. we check everywhere before using wallet.wallet if its not None.

Reply:
Option 1 would be a lot more elegant. Is it in the scope of this PR?

Reply:
00dba54a
pushed changed with option 1, add a couple of ignore, we really should do a followup PR fixing those ignore types up